package.json

```diff
@@ -13,15 +13,18 @@
   "scripts": {
     "prepare": "is-ci || husky install",
     "build": "tsc -p tsconfig.build.json",
+    "prebuild-release": "yarn code-quality:check",
```

Comment:
is this used anywhere?

Reply:
yes. Feel free to read: https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts to get better knowledge. But in short whenever you run `yarn build-release` a `prebuild-release` is run